core/__init__.py

```diff
@@ -6,4 +6,5 @@
 import core.state
 
 # RoveComm node, must be initialized before it can be used.
-rovecomm: RoveComm = None
+rovecomm: RoveComm
+
```

Comment:
I am not totally sure the reason for this change. It is very minor and seems out of place for this feature, but can be left in if it isn't part of a larger set of fixes. However, it appears the comment above should be updated as you are no longer initializing rovecomm.